src/components/common/buttons/CustomButton.js

```diff
@@ -58,9 +58,6 @@ const mapPropsToStyles = ({ theme, compact }) => ({
     paddingTop: 1,
     letterSpacing: 0,
   },
-  contentStyle: {
-    letterSpacing: 0,
```

Comment:
I can't remember the reason for this rule. So it's Ok for me.

Reply:
that's a change by dario

Reply:
yes, that content was in the past text, now it is not so it is ok to remove this